index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '@segment/in-eu' {
```

Comment:
You don't need the `declare module '@segment/in-eu'`, e.g: https://github.com/segmentio/remote-service-client/blob/6df17fed3db1799fb0358b64ca17337a4f781785/types/index.d.ts

Reply:
TIL!